packages/dd-trace/src/plugins/util/git.js

```diff
@@ -58,22 +58,34 @@ function getCommitsToUpload (commitsToExclude) {
 
 // Generates pack files to upload and
 // returns the ordered list of packfiles' paths
+function execGitPackObjects (targetPath, commitsToUpload) {
+  return execSync(
+    `git pack-objects --compression=9 --max-pack-size=3m ${targetPath}`,
```

Comment:
This triggers a security warning. Any way to do this while avoiding the warning?

Reply:
I'll see what I can do, but the received commits should already be sanitized